site/src/cluster.js

```diff
@@ -1,5 +1,6 @@
 import { Cluster } from '@nftstorage/ipfs-cluster'
 import { cluster } from './constants.js'
+import { CID } from 'multiformats'
```

Comment:
can we just use the cid string ?

Reply:
We need CIDs to assemble an IPLD node. I could move the the CID string parsing to the other module, if that is what you want but CID is needed one way or the other.

@hugomrdias please let me know if this is ok, or if you want me to move the parsing to other module.

Reply:
i would prefer to remove this here but its not a blocker